x-pack/plugins/secops/public/components/page/network/network_dns_table/columns.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import numeral from '@elastic/numeral';
+import { isNil } from 'lodash/fp';
+import React from 'react';
+
+import { NetworkDnsFields, NetworkDnsItem } from '../../../../graphql/types';
+import { escapeQueryValue } from '../../../../lib/keury';
+import { networkModel } from '../../../../store';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { defaultToEmptyTag, getEmptyTagValue } from '../../../empty_value';
+import { Columns } from '../../../load_more_table';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import * as i18n from './translations';
+
+export const getNetworkDnsColumns = (
+  startDate: number,
+  type: networkModel.NetworkType
+): Array<Columns<NetworkDnsItem>> => [
+  {
+    field: `node.${NetworkDnsFields.dnsName}`,
+    name: i18n.REGISTERED_DOMAIN,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: true,
+    render: dnsName => {
+      const id = escapeDataProviderId(`networkDns-table--name-${name}`);
```

Comment:
Looks like this id isn't always unique. Steps to recreate:
- Click `Load More`
- Sort table column
- Change `Row Count` from `10` to `5`

![top_dns_draggable_crash](https://user-images.githubusercontent.com/2946766/54459292-eacd4e00-472b-11e9-9898-dc26d8968333.gif)